test/proxyservice/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.0'
+
+services:
+  secretless:
+    image: secretless-broker
+    volumes:
+      - ./secretless-with-errors.yml:/secretless.yml
+
+  secretless-dev:
+    image: secretless-dev
+    command: sleep 999d
```

Comment:
Instead of `sleep 999d`, can we use a command that runs forever? e.g.:
```
    command: tail -f /dev/null
```

Reply:
Yeah there's really no good way to really wait forever in bare v3 shell ([long thread here](https://unix.stackexchange.com/questions/42901/how-to-do-nothing-forever-in-an-elegant-way/42905#42905)) and we have [generally used](https://github.com/cyberark/secretless-broker/search?q=sleep+999d&unscoped_q=sleep+999d) `sleep` since it's more clear about hat's happening. Up to you if you want this changed.

Reply:
Ah, no worries, let's stick with the `sleep` convention.